### DISCUSSION TITLE
sensors not available?

### DISCUSSION THREAD

**Discussion by Cold-Lemonade** [mrlt8/docker-wyze-bridge#1063] [Q&A]
I am running docker-wyze-bridge v2.6.0 as an add-on in HA. My MQTT integration shows my Wyze Cam v3, but all four sensors are listed as unavailable. For example, the camera in my kitchen has a sensor called ```camera.wyze_cam_kitchen_cam``` but it is listed as unavailable. If I click on it, I get a message saying: "This entity is no longer being provided by the mqtt integration. If the entity is no longer in use, delete it in settings."

Can someone please help me understand what is going on?

**Comment by mrlt8** [comment]
Is the add-on running? can you try removing the sensors that are unavailable and see if they return?

**Reply by Cold-Lemonade** [reply]
Yes, I am running the add-on in HA. I've tried deleting the sensors and restarted HA, but those sensors do not return. I am not sure what to do. The original message seemed to indicate that those sensors are no longer supported by MQTT.

**Reply by mrlt8** [reply]
Do you remember what those sensors were? A couple of the MQTT entities were renamed, so it could be that the old names were still cached in HA and showing as unavailable?

**Comment by felafel** [comment]
I have been getting this message for ages [6+ months]; "This entity is no longer being provided by the mqtt integration. If the entity is no longer in use, delete it in settings.".

I thought this add-on was broken.  I have very recently changed the port from 8554 to 8557 and that did fix the ability to see the cameras when I open the Web UI.  However it still does not fix seeing the previews on my main page.

Edit: I also get "Lost connection to the bridge..." when I am in the Web UI page.  If I can help with debugging please let me know how.

Edit2: I get this in the log --

[WyzeBridge] 🏠 Home Assistant Mode
[WyzeBridge] WARNING: invalid escape sequence '\:'

🚀 DOCKER-WYZE-BRIDGE v2.6.0 X86_64 2.7.0 BUILD [2024-01-11t04:20:13.342z] bbacb0f

Note that I would like to update to v2.7 and at the bottom of the Web UI page there is an update symbol that tells me that there is an update available to v2.7.  I do not see a way to update in Home Assistant Settings or in the Web UI page.
